### PR TITLE
Query subscriptions

### PR DESCRIPTION
**Pull request by Jlrine2**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `POST /subscriptions` requests may now include a `validate_only` key which when set to `true` will not add the subscription
 to the database but still validate it.
 - in `POST /subscriptions` requests, `search_parameters` and `job_specification` are now included under `subscription`
+- `GET /subscriptions` requests now include query parameters
+  - `name` gets only subscriptions with the given name
```
Comment (jhkennedy) [inline CHANGELOG.md:13]: ...with the given project name? Job name?

File: apps/main-cf.yml [hunk 2]
```diff
@@ -254,5 +256,7 @@ Resources:
           KeySchema:
             - AttributeName: user_id
               KeyType: HASH
+            - AttributeName: creation_date
+              KeyType: RANGE
           Projection:
             ProjectionType: ALL
```
Comment (jhkennedy) [inline apps/main-cf.yml:262]: trailing new line?

**Review by jhkennedy — APPROVED** [review]
Looks good to me! Just two super minor things

File: apps/main-cf.yml (hunk 2 above)
Comment (Jlrine2) [reply apps/main-cf.yml:262]: didn't have one before

File: CHANGELOG.md (hunk 1 above)
Comment (Jlrine2) [reply CHANGELOG.md:13]: I'm leaving it as `name` because that is the name of field in both subscriptions and jobs

Comment (jhkennedy) [reply CHANGELOG.md:13]: I think you should probably specify that it's in the job specification, not a subscription name

**Pull request merged**